### ISSUE TITLE
lack of placeholder in logs in serviceconfig.go

### ISSUE THREAD

**Issue by inspuradmin** [inspursoft/board#259]
In GitLab by @luotian-github on Dec 4, 2017, 17:13

main.go 58:
```
if subPath != "" {
```

the  logic expression 
```
defaultProject         = "library"
subPath := defaultProject
if subPath != "" {
```
is always true

**Comment by inspuradmin** [comment]
In GitLab by @luotian-github on Nov 8, 2019, 07:42

Fixed in PR#709

**Comment by inspuradmin** [comment]
In GitLab by @sokril on Nov 8, 2019, 08:15

closed
